src/site/content/en/learn/css/media-queries/index.md

```diff
@@ -0,0 +1,786 @@
+---
+title: Media Queries
+description: >
+  In this module learn the ways you can style backgrounds of boxes using CSS.
+audio:
+  title: 'The CSS Podcast - 036: Page Media'
+  src: https://traffic.libsyn.com/secure/force-cdn/highwinds/thecsspodcast/TCP032_v2.mp3?dest-id=189156
+  thumbnail: image/foR0vJZKULb5AGJExlazy1xYDgI2/ECDb0qa4TB7yUsHwBic8.png
+authors:
+  - lozandier
+date: 2021-12-06
+---
+
+*CSS Media queries allows styles or stylesheets to be conditionally applied based on specific media conditions. Find out how to use them effectively within your stylesheets in this module.*
+
+Suppose you had a group of cards that you had arranged to be in one column on a small viewport, but wanted them to line-up to two & three columns when more space is available for the cards.
+
+
+With CSS Media Queries you can solve this problem:
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'YzryWpw',
+  theme: 'dark',
+  height: 700,
+  tab: 'css, result'
+} %}
+
+{% YouTube "JLRFF56_S3U" %}
+
+Because the Web is agnostic, there became a need across standardized Web languages for Web developers to be able inquire whether or not a set of criteria applies to a medium. The result of that was the [Media Query]  standard, a normalized means to write a query with a `true` or `false` result whether certain media conditions hold true for a particular device the Web is being consumed on.
+
+CSS styles are applied to documents that can be consumed in a medium other than browsers; accordingly media queries are a handy way for Web creatives to conditionally apply styles to fix, enhance, or maximize a document's usefulness accounting for particular media conditions applicable
+
+In other words, keeping in mind `media` is plural for medium, CSS media queries allow you to apply styles originating from blocks of CSS or a stylesheet depending on medium-related conditions applicable to the device a user is viewing a particular document with. These media-related conditions include the capabilities you require a medium to have & even the users preferences being actively accommodated by the medium.
+
+In CSS you apply a media query using either the  `@media`  or `@import`  [at rules](https://thecsspodcast.libsyn.com/031-rules)  followed by the [**media query**](l).
+
+`@media` is used to conditionally apply styles based on a media query while `@link` conditionally loads a stylesheet based on a media query.
```

Comment:
Did you meant the `<link>` element here? I searched around for a `@link` at rule, but wasn't able to find anything. If you meant `<link>`, that element only conditionally loads a stylesheet if its `media` attribute (which contains a media query) applies. Even then, I believe some browsers will download _all_ styles regardless of of the `media` attribute value, but will degrade the priority of unmatching `media` conditions for those stylesheets to "low".